db/mockdata.py

```diff
@@ -0,0 +1,159 @@
+"""
```

Comment:
I could not find a place to keep these classes so i created a temporary file.
These classes are needed only by the **api_compat** so merging them into existing files is also not a good idea. I can move them to **api_compat.py** or **api_tools.py** to make sure that the api_compat is modular and its bits are not spread across the whole app.

I would love to here any suggestions from you.